### DISCUSSION TITLE
9VAC Power Adapter Requirement

### DISCUSSION THREAD

**Discussion by algovera** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#65] [Q&A]
Is there a min amp requirement for the 9VAC adapter?

I read there is a 500mA requirement for 220VAC > 12 VAC

Not sure if the same for 9VAC

**Comment by CircuitSetup** [comment]
I would recommend using a 1000mA transformer if possible.

**Reply by wifixcort** [reply]
Why would you need so much current? Even 750mA is too much, 500mA for sourcing ICs and a few uA for sensing is needed or I'm wrong?

**Reply by missmah** [reply]
Because the main transformer also powers the board itself.

**Comment by caladeira** [comment]
Hi to all,

Regarding the power requirements, I'm monitoring a three phase system (3x
230v) and for the first phase I'm going to use a 9VAC 1A transformer.

If I had understood correctly, the other two phases will need a 9VAC
transformer but they can have a small current capacity.

Can I use for this two phases encapsulated transformers with the specs:
9VAC 1.8VA (200ma)
https://www.tme.eu/Document/fc9cac9a71bdba148b73e1dbfe4a0516/BVEI3022001.pdf

Thank you.

**Reply by CircuitSetup** [reply]
Yes, you can. You'll just need to calibrate the meter to them using a volt meter.

**Comment by alexruffell** [comment]
I made a power supply and voltage reference board to use with the circuitsetup energy meter. I used a 12VAC transformer to power the board and a 9VAC transformer to provide a voltage reference for the other phase. 

![20220805_105141](https://user-images.githubusercontent.com/4006944/183114813-3ed3288a-a44c-45d2-a651-6d50492a720e.jpg)

In the image above you may note that the larger transformer has 2 windings 6VAC each. They are connected in series (per the manufacturer's instructions) to obtain a 12VAC. The transformer is 14VA.

All references I find say not to input more than 12VAC however when I measure the output of both my transformers they are close to 15VAC which is quite typical for unregulated power supplies. A quick test shows everything working correctly but I wonder whether this pushes something out of spec and will lead to premature failure.

Is the 12VAC limit referring to what the nominal voltage spec of the transformer? If so then I assume the no load 15VAC measured on the transformer secondary should be fine. Please confirm, I don't want to fry >$100 of your products...

**Reply by ICPete** [reply]
Hi Alex,
Did you ever determine an answer to this question?
If so, do you have design data available for this custom board?
I use Kicad, but even a PDF schematic would be great.

**Reply by alexruffell** [reply]
@ICPete Kicad is on my list to learn but I don't have the time right now... I used excel and a laser printer to print on special paper that transfers the toner to the PCB with heat from an iron. Lol, very rudimentary! I switched to a 10V AC transformer so I have to redesign, in Kicad this time, the board as the 2 secondary windings are now in parallel not series. Somewhere in another issue thread you will find details on the max voltage the IC input can accept before being out of spec. I am on my phone so I can't look easily but I think the max voltage was 750mV and there is a 20:1 divider so max voltage was 15V and I was supplying 0.5V above that.

**Reply by ICPete** [reply]
OK thanks Alex.

**Reply by tsrattan** [reply]
alexruffell. Could you let me know if you are able to use your setup?

**Comment by ralbright-wa** [comment]
I am looking to use my Energy Monitoring In my fifth wheel. My Power WatchDog surge already tells me the voltage and wattage loads of the 2 main legs for my 50 amp connection. Near my power center I do not have an accessible 110VAC outlet. I would like to power the system here from my DC battery setup using a small 100W inverter and then plug the 9VAC into that. Will this work? Or is there a more elegant solution utilizing DC power to power this setup?
